.NET/Microsoft.Recognizers.Text.Sequence/SequenceRecognizer.cs

```diff
@@ -105,6 +110,18 @@ public IModel GetHashtagModel(string culture = null, bool fallbackToDefaultCultu
             return GetModel<HashtagModel>(Culture.English, fallbackToDefaultCulture);
         }
 
+        public IModel GetQuotedTextModel(string culture = null, bool fallbackToDefaultCulture = true)
+        {
+            if (culture != null && (
+           culture.ToLowerInvariant().StartsWith("zh-", StringComparison.Ordinal) ||

```

Comment:
This existed in other types as there were only two models. But in your case there will be a bunch, right? Or also only two main ones that cover all cultures?